packages/release/ldconfig-service.conf

```diff
@@ -0,0 +1,4 @@
+[Unit]
+# Run `ldconfig.service` after `tmpfilesd`, since `ldconfig` will attempt to
+# read files from `/etc/ld.so.conf.d`, which are created by `tmpfilesd`
+After=systemd-tmpfiles-setup.service
```

Comment:
Let's put this in `glibc` as well; I think that's more clear since it's the use of `ld.so.conf` via tmpfiles that creates this ordering requirement.